libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -104,6 +104,21 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MIS_ITEMS",    7,     AP_Arming, _required_mission_items, 0),
 
+    // @Param: BARO2GPS
+    // @DisplayName: Compare Baro Alt to GPS altitude
+    // @Description: Strict barometer validation by comparing BARO-based Altimeter to GPS altitude
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Advanced
+    AP_GROUPINFO("BARO2GPS",     8,     AP_Arming, _baro2gps, 0),
+
+    // @Param: ARSPD_MAX
+    // @DisplayName: Maximum valid PreArm airspeed
+    // @Description: Maximum airspeed value to pass airspeed arming check. Make sure airspeed is loosely covered. Set 0 to disable checking.
+    // @Range: 0.1 20
+    // @Units: m/s
+    // @User: Advanced
+    AP_GROUPINFO("ARSPD_MAX",    9,     AP_Arming, _arspd_max, 0),
```

Comment:
Probably worth making this an `AP_GROUPINFO_FRAME` so that it can be excluded from copters. IE

`AP_GROUPINFO("ARSPD_MAX",    9,     AP_Arming, _arspd_max, 0, AP_PARAM_FRAME_PLANE | AP_PARAM_FRAME_ROVER),`

Reply:
Done, Please review